app/services/github_service.rb

```diff
@@ -4,6 +4,11 @@ def user_repos(user_token)
     JSON.parse(response.body, symbolize_names: true)
   end
 
+  def user_followers(user_token)
```

Comment:
Since I'll have code almost exactly like this, might be a good place to refactor in the future